shub/utils.py

```diff
@@ -136,12 +137,12 @@ def patch_sys_executable():
         # it so the system-wide Python installation uses its own library path
         # (this is particularly important if the system Python version differs
         # from the Python version that the binary was compiled with)
-        orig_lib_path = os.environ.pop(b'LD_LIBRARY_PATH', None)
+        orig_lib_path = os.environ.pop('LD_LIBRARY_PATH', None)
```

Comment:
Is there a good reason for making this a unicode instead of bytes literal? This will [break the Windows binary](https://github.com/scrapinghub/shub/pull/148#issuecomment-173417050).